xla/stream_executor/dnn.h

```diff
@@ -1706,6 +1714,9 @@ class DnnSupport {
       std::optional<dnn::TensorDescriptor> dscale_descriptor,
       std::optional<dnn::TensorDescriptor> dbias_descriptor);
 
+  virtual absl::StatusOr<std::unique_ptr<DnnGraph>> DeserializeGraph(
+      absl::string_view) const = 0;
```

Comment:
I think you need to provide default that returns error, as I'm pretty sure today with earlier cudnn versions it will fail at link time (or compile time)

Reply:
Done.